calico-enterprise/multicluster/set-up-multi-cluster-management/helm-install/create-a-management-cluster-helm.mdx

```diff
@@ -156,11 +155,52 @@ To install a {{prodname}} [management](create-a-management-cluster-helm#value) c
   watch kubectl get tigerastatus
   ```
 
+
+#### Update the ManagementCluster address.
+
+1. Export the service port number
+
+  ```bash
+  export EXT_NODE_PORT=<your-load-balancer-external-node-port>
```

Comment:
Probably best to avoid the term node port as it's not really what this port is, would suggest:

```suggestion
  export EXT_LB_PORT=<your-external-load-balancer-port>
```